Composer/packages/tools/language-servers/language-generation/package.json

```diff
@@ -8,7 +8,7 @@
     "build:demo": "cd demo && tsc --build tsconfig.json",
     "prepublishOnly": "npm run build",
     "clean": "rimraf lib demo/dist",
-    "start": "cd demo && ts-node ./src/server.ts",
+    "start": "cd demo && set NODE_ENV=development&& ts-node ./src/server.ts",
```

Comment:
- insert a space between `NODE_ENV=development` and `&&`
- also you might want to use the [cross-env](https://github.com/kentcdodds/cross-env) package to set this environment variable because `set <variable>=<value>` is not cross-platform